create_module.rb

```diff
@@ -0,0 +1,52 @@
+module CreateElement
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
```

Comment:
- [ ] You are using print and gets.chomp at a lot of places means that you are repeating the code so your code is not DRY. Kindly make a separate function for taking inputs 👍 to reuse that everywhere you wanna get inputs 😊

Hint: See the function below ⬇️
```
def inputs(show)
  outputs = []
  show.each do |s|
    print s
    outputs.push(gets.chomp)
  end
  outputs
end
```